src/CVNBot/CVNBot.csproj

```diff
@@ -20,26 +20,20 @@
     <ExternalConsole>false</ExternalConsole>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="log4net, Version=2.0.8, Culture=neutral">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\log4net.dll</HintPath>
-    </Reference>
-    <Reference Include="Meebey.SmartIrc4net, Version=1.1, Culture=neutral">
```

Comment:
When I inspect the current embedded DLL for SmartIrc4net, VS2019 says it is actually version 0.4.5.0. But I'm not sure if I'm using interpreting this correctl and/or whether that is the one msbuild is indeed using.

Reply:
This is correct. If you download the version 1.1 of the NuGet package and peek inside the .dll you'll notice that the version is 0.4.5.0 and the AssemblyInformationalVersion is 1.1.
![image](https://user-images.githubusercontent.com/679326/84336543-562ee700-ab4c-11ea-8f7e-32aed0b4631d.png)